include/flatbuffers/verifier.h

```diff
@@ -25,15 +25,27 @@ namespace flatbuffers {
 // Helper class to verify the integrity of a FlatBuffer
 class Verifier FLATBUFFERS_FINAL_CLASS {
  public:
+  struct Options {
+    uoffset_t max_depth = 64;
+    uoffset_t max_tables = 1000000;
+    bool check_alignment = true;
+    bool check_nested_flatbuffers = true;
+  };
+  Verifier(const uint8_t *const buf, const size_t buf_len, Options opts)
```

Comment:
This should be: 
`const Options& options = Options{}`

Reply:
If we do, the compiler will think there's an ODR violation (with the legacy constructor when there are no default arguments). I think we should just migrate away in the next major release